components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/FontResources.kt

```diff
@@ -32,4 +33,20 @@ expect fun Font(
     resource: FontResource,
```

Comment:
It is late and not for this PR, but `Font` function isn't by Compose guidelines.

If it returns a value, it should start with lower case.

Reply:
It's done to mirror the Android approach:
```kotlin
val firaSansFamily = FontFamily(
    Font(R.font.firasans_light, FontWeight.Light),
    Font(R.font.firasans_regular, FontWeight.Normal),
    Font(R.font.firasans_italic, FontWeight.Normal, FontStyle.Italic),
    Font(R.font.firasans_medium, FontWeight.Medium),
    Font(R.font.firasans_bold, FontWeight.Bold)
)
```

https://developer.android.com/develop/ui/compose/text/fonts

Reply:
`FontFamily` and `Font` aren't Composable functions, they are data classes that have a link to a resource. This link is resolved later, when it is used inside Composable function.

If we want to maintain the Compose API and maintain Compose guidelines at the same time, we need to do the same.